SpringProject/src/it/java/com/gradproject2019/ConferenceControllerIT.java

```diff
@@ -92,7 +89,7 @@ public void shouldReturn404WhenIdDoesNotExist() throws URISyntaxException {
         Assert.assertEquals(404, responseConference.getStatusCodeValue());
     }
 
-    @Test
+    @Test  @Ignore //CANNOT FUNCTION WHILE NOT USING DOCKER (CONTAINERISATION)
```

Comment:
Why did you write a test if you cannot test it and it's not working at the moment?
If you've decided to commit this code then it should work, otherwise it's a dead code and should be removed from the project.